### ISSUE TITLE
MapPosition needs to be updated

### ISSUE THREAD

**Issue by CarloSegat** [yovanoc/cookiebot#112]
If I run the app I get the following error: Missing field 'AllowPrism' (Bool) in class 'MapPosition'

**Comment by yovanoc** [comment]
Yeah of course you are right, I spent all my freetime on my other (CookieTouch) project and I have no time to update this, but it will be soon

**Comment by CarloSegat** [comment]
Hi Yovanoc, I do not mean to pressure you, thank you very much for your work.
I'm having a look at why is not working but I doubt I'll figure how to fix it :) 
My wildest guess is that the api for achievements has changed.

**Comment by yovanoc** [comment]
Yeah a lot of changes since the 2.45 version
